LeaderboardBackend/AppConfig.cs

```diff
@@ -4,6 +4,7 @@ namespace LeaderboardBackend;
 
 public class AppConfig
 {
+    public static Uri BASE_PATH = new("https://leaderboards.gg");
```

Comment:
Also, this shouldn't be called BASE_PATH because it's actually the URL of the frontend isn't it?

Reply:
Yeah it's probably not the best name. Maybe `BASE_URL`?

And I've made it static because I don't see why it should be an instance member, unless injecting AppConfig into the service is preferable.

Reply:
Seeing as it's not the base of _this_ app necessarily, I think WEBSITE_URL or FRONTEND_URL would be preferable.

Reply:
I agree with Tedder on naming.

----
Don't make it static, it sucks for testing.
`AppConfig` is already injectable as a `IOptions<AppConfig>`.

Also don't set a default value, it should be set when setting up the app.
A default value should be good for most cases, however here it's only good in the case of a specific environment (lbgg's production).
You can add a `.NotNull()` validation for it.

Do add it to the `example.env` though,  
Setting it to the frontend default local url is nice. `AppConfig__WebsiteUrl=http://localhost:3000` (I don't remember the actual default port) 

----
No need to do that right now, but with this new config we can allow cross origin for this URL instead of relying on the `AllowOrigins` config, and it can thus be made optional.